lib/core/src/Cardano/Wallet.hs

```diff
@@ -1623,6 +1632,63 @@ forgetTx ctx wid tid = db & \DBLayer{..} -> do
   where
     db = ctx ^. dbLayer @s @k
 
+-- | Given a LocalTxSubmission record, calculate the slot when it should be
+-- retried next.
+--
+-- The current implementation is really basic. Retry every 10 slots.
+scheduleLocalTxSubmission :: LocalTxSubmissionStatus tx -> SlotNo
+scheduleLocalTxSubmission st = (st ^. #latestSubmission) + 10
```

Comment:
So that's about half the block frequency on mainnet, which would means effectively retrying on every block. Maybe a little _excessive_ for exchanges & al, but I see no harm doing this otherwise.

Reply:
Fixed in #2611